tests/test_migration_steps.py

```diff
@@ -0,0 +1,176 @@
+import shutil
```

Comment:
Overall these tests are pretty straightforward, I think having a doc comment on each one explaining what it's asserting would still be useful

Reply:
Added a bunch more - let me know if they make sense!